src/Form/FormMapper.php

```diff
@@ -136,10 +136,21 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
                 $options['label'] = $this->admin->getLabelTranslatorStrategy()->getLabel($name, 'form', 'label');
             }
 
+            if (isset($options['label_translation_parameters'])) {
+                $fieldDescription->setOption('label_translation_parameters', $options['label_translation_parameters']);
+                unset($options['help']);
+            }
```

Comment:
I don't think we need this, all options are set by default to the `$fieldDescription`.

There is special behaviour for `label`, `template` etc because the data is saved in a `$label`, `$template` property instead of the `$options` property (don't really know why btw...).

Reply:
BTW it currently introduce a bug since you're unsetting `$options['help']`.